selfdrive/car/chrysler/values.py

```diff
@@ -129,12 +129,13 @@ class ChryslerCarInfo(CarInfo):
   }],
 }
 
-FW_VERSIONS = {
-  CAR.PACIFICA_2019_HYBRID: {
-    (Ecu.hcp, 0x7e2, None): [],
-    (Ecu.abs, 0x7e4, None): [],
-  },
+# Ecus added for data collection
+EXTRA_ECUS = {
+  (Ecu.hcp, 0x7e2, None),  # manages transmission on hybrids
+  (Ecu.abs, 0x7e4, None),  # alt address for abs on hybrids
+}
```

Comment:
let's formalize this a bit more into a generic FPv2 config

Reply:
How does this look now?